src/lambdas/confirmDownload/auditTemporaryS3LinkCreated.ts

```diff
@@ -1,26 +1,29 @@
 import { sendSqsMessage } from '../../sharedServices/queue/sendSqsMessage'
 import { currentDateEpochSeconds } from '../../utils/currentDateEpoch'
 import { getEnv } from '../../utils/getEnv'
-
-export const auditTemporaryS3LinkCreated = async (zendeskId: string) => {
+import { logger } from '../../sharedServices/logger'
+export const auditTemporaryS3LinkCreated = async (
+  zendeskIdInternal: string
+) => {
   try {
+    logger.appendKeys({ zendeskId: zendeskIdInternal })
```

Comment:
You don't need to rename the variable here, because typescript/javascript lets you use keys which have implicit names. So you can just put 

```suggestion
    logger.appendKeys({ zendeskId })
```

Reply:
fixed